docs/dev/system_keyspace.md

```diff
@@ -215,6 +217,9 @@ Only tables which use tablet-based replication strategy have an entry here.
 `tablet_count` is the number of tablets in the map.
 `table_name` is the name of the table, provided for convenience.
 
+`resize` is the resize decision type, which can be one of: `merge`, `split` or `none`.
+`resize_phase` is the current phase for a table, to distinguish stale to up-to-date resize decisions.
```

Comment:
This doesn't say anything. What's a resize phase? What's the meaning of the value "44"?

Reply:
?

Reply:
This is outdated.